onboaring.md

```diff
@@ -0,0 +1,345 @@
+In the example Warehouse expects to find asset files in the folder `./dist`, named `output[.min].js` or
+`output[.min].css` respectively.
+
+```ini
+[files]
+dev = ['dist/ouput.js', 'dist/output.css']
+test = ['dist/ouput.js', 'dist/output.css']
+prod = ['dist/ouput.min.js', 'dist/output.min.css']
+```
+> We plan to do all configuration from this file in the future.
+> This will allow us to expose more features and make setup easier.
+
+From this example you can discern that uploaded files are configurable per environment and file paths are
+determined relative to the root of your module. The `dist` directory is not an arbitrary choice, `warehouse.ai` will need the `dist` directory to explicitly exist so it knows which files to serve.
+
+Finally, if you are using `webpack`, you will need a `webpack.config.js` in the root directory. Make sure that files in your `wrhs.toml` match the ouput of your `webpack.config.js`. It is important to note that all `warehouse.ai` is doing is to call `webpack` in this case, thus all of your configuration must live within this file (not as command line arguments). 
+
+```js
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'output.js'
+  }
+};
+```
+#### Extract Config
+
+COMING SOON: You will be able to extract your warehouse.ai configuration to keep it in one place by using [extract-config]. 
+
+### 4. Publishing your module
+
+If you followed above steps carefully, your module should be ready to be published to Warehouse. Let's publish your
+first module to Warehouse!
+
+Publish your package as you normally would:
+
+```sh-session
+npm version <newversion> 
+
+npm publish
+```
+
+This will trigger a build in Warehouse and publish the module to the `write` endpoint you specified. Initially the build of this version
+will only be available in `DEV`. Subsequently the files are only uploaded to the `DEV` CDN as well. A hash is generated
+from the file content that the browser will use to do caching based on file content. Warehouse provides
+various mechanics to retrieve the build files. We'll [dive into the mechanisms
+later](#using-build-output).
+
+
+#### Monitoring build progress
+
+To check the status of the build process you can either use the [Warehouse UI][ui] or
+[CLI](cli).
+Once you have `Warehouse UI` set up, it will show the
+build progress for each environment. Wait for the `DEV` build to complete.
+
+#### Promote your module to TEST/PROD
+
+After the initial publish, the build will only be available in the `DEV` environment. Assuming the bundle
+works as expected, it can be promoted to `TEST` and/or `PROD`. Promoting your module's bundle to a specific
+environment can be done through `npm dist-tag`. For example, to promote your bundle to `TEST` run:
+
+```
+npm dist-tag add [module-name]@[semver] test --reg=https://wherever-you-deployed-warehouse.ai
```

Comment:
```suggestion
npm dist-tag add [module-name]@[semver] test --registry=https://wherever-you-deployed-warehouse.ai
```

We should spell out all the commands in docs, even when short forms exist. That way anyone not familiar with the short forms can still understand the docs and what the arguments are for.